### PR TITLE
Documentation of json.mset.md

### PR DESCRIPTION
**Pull request by adrianoamaral**
Command reference details of the new JSON.MSET

### CODE REVIEW

File: docs/commands/json.mset.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+## Required arguments
+
+<details open><summary><code>key</code></summary>
+
+is key to modify.
+</details>
+
+<details open><summary><code>path</code></summary>
+
+is JSONPath to specify. Default is root `$`. For new Redis keys the `path` must be the root. For existing keys, when the entire `path` exists, the value that it contains is replaced with the `json` value. For existing keys, when the `path` exists, except for the last element, a new child is added with the `json` value.
```
Comment (LiorKogan) [inline docs/commands/json.mset.md:19]: Default is root `$`
- How can we have a default? This is a required argument.

File: docs/commands/json.mset.md [hunk 2]
```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
+
+JSON.MSET is atomic, so all given keys are set at once or not. It is not possible for clients to see that some of the keys were updated while others are unchanged.
+
+Important, JSON is a hierarquical structure if you change a value in specific path, subsequent nested values will be affected and may be unchanged
```
Comment (LiorKogan) [inline docs/commands/json.mset.md:5]: <note><b>Note:</b>
A JSON value is a hierarchical structure. If you change a value in a specific path - nested values are affected. 
</note>

File: docs/commands/json.mset.md [hunk 3]
```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
+
+JSON.MSET is atomic, so all given keys are set at once or not. It is not possible for clients to see that some of the keys were updated while others are unchanged.
```
Comment (LiorKogan) [inline docs/commands/json.mset.md:3]: JSON.MSET is atomic, hence, all given additions or updates are either applied or not.

File: docs/commands/json.mset.md [hunk 4]
```diff
@@ -1 +1,60 @@
...
+127.0.0.1:6379> JSON.GET doc2 $
+"[{\"f\":{\"a\":3]"
+127.0.0.1:6379> JSON.GET doc3
+"{\"f1\":{\"a\":3},\"f2\":{\"a\":3}}"
+{{< / highlight >}}
+</details>
+
+## See also
+
+`JSON.SET` | `JSON.MGET` | `JSON.GET`
```
Comment (LiorKogan) [inline docs/commands/json.mset.md:55]: Please add JSON.MSET to the JSON.SET page as well.

File: docs/commands/json.mset.md [hunk 5]
```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
```
Comment (LiorKogan) [inline docs/commands/json.mset.md:1]: Set or update one or more JSON values according to the specified `key`-`path`-`value` triplets.

**Review by LiorKogan — CHANGES_REQUESTED** [review]

File: docs/commands/json.mset.md (hunk 3 above)
Comment (adrianoamaral) [reply docs/commands/json.mset.md:3]: OK

File: docs/commands/json.mset.md (hunk 1 above)
Comment (adrianoamaral) [reply docs/commands/json.mset.md:19]: It's optional

File: docs/commands/json.mset.md (hunk 4 above)
Comment (adrianoamaral) [reply docs/commands/json.mset.md:55]: Done in a new PR

**Review by adrianoamaral — COMMENTED** [review]
comments addressed

File: docs/commands/json.mset.md [hunk 6]
```diff
@@ -1 +1,60 @@
...
+</details>
+
+<details open><summary><code>value</code></summary>
+
+is value to set at the specified path
+</details>
+
+## Optional arguments
+
+<details open><summary><code>path</code></summary>
```
Comment (LiorKogan) [inline docs/commands/json.mset.md:24]: I don't think it is optional.

**Review by oshadmi — APPROVED** [review]

**Pull request merged**